README.md

```diff
@@ -15,6 +15,16 @@ This is a [Node.js](https://nodejs.org/en/) module available through the
 $ npm install express-session
 ```
 
+## Debugging
+
+```sh
+$ export DEBUG=express-session
```

Comment:
I have had this in a module before and it resulted in endless PRs for al the different methods to set environment variables on the different OSes / shells. I would prefer not to go through that again and end up with a giant list of these :) If it is possible, just state what our namespace is (`express-session` and point the user to the `debug` module on how to turn on debugging for a given namespace.

Reply:
Would it suffice to give a command that suggests running your project while setting the DEBUG flag aka `DEBUG=express-session node index.js`? I'm not 100% but I think that is an appropriate cross platform way to set the ENV var?

Reply:
Sadly that syntax does not work on `cmd.exe`, nor on PowerShell, both the main Windows shells... :(

Reply:
@dougwilson - my motivation is to document critical, yet missing information that aids self-diagnosing issues of using this module. Reading it multiple times, I don't see any misleading, confusing or redundant phrases, nor any incomplete info that will / may invite future bike-shedding.

I would like to retain the changes. I see you did not use the red button, so I treat your comment as a suggestion that is free to leave out, not a blocker.

Reply:
The issue is in the past, this just leads to endless issues being opened reporting the following error:

![image](https://user-images.githubusercontent.com/67512/73587556-9047f800-448b-11ea-8b4d-7e7e0668ef0b.png)

Reply:
And yes, you are 100% correct, I did not "request" changes, and only made a comment :) That was intentional... I am not saying there _needs_ to be a change. I was thinking about where we have left something similar and it was http://expressjs.com/en/starter/generator.html … after countless PRs from users "fixing" the DEBUG command. Maybe just copy that structure since we haven't had a PR in a while around that. I think the current structure leads to two issues:

1. users read top-to-bottom, and will try to run a command given before reading down below. Only below it says that doesn't work on Windows. On expressjs.com what is works on is disclosed before the command.
2. The text says "use set in Windows" but I can tell you from experience users … have no idea that that means (this is not unique to windows; export has the same thing). It would seem to either say "use export" and "use set" or give full examples for both.

Reply:
`Setting this property prior to the run (use 'set' in Windows)` - the first line in my change has this, that should clarify, and work I guess?

Reply:
may be we all 3 talked almost at the same time, so I am not sure about the ordering of the above conversation - who is asking, and who is answering to whom! :)

Reply:
ok , I understand your reasons. Such users (who cannot reasonably follow the document) can also copy-paste the dollar symbol, and `app.js` from the phrase `$ DEBUG=express-session node app.js` while there shell prompt and app name might have been something else?

my take would be to balance between readability and usability, while accepting reasonable changes (in the future) and provide explanation to the rest.

Reply:
Right, and that's why I suggest we just copy what is on expressjs.com page I shared, as that seems to have calmed down the PRs. I really don't want to land a PR that is _known_ to create a cascade of more PRs to deal with.